resizer/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Resizer",
+  "id": "994314547752599560",
+  "api": "1.0.0",
+  "main": "code.js",
+  "ui": "ui.html",
```

Comment:
Since both of these are `parameterOnly` now, I think we can remove this and the corresponding `ui.html` file.